scripts/src/precheck/submission.py

```diff
@@ -73,6 +88,53 @@ def register_chart_info(self, category, organization, name, version):
     def get_owners_path(self):
         return f"charts/{self.category}/{self.organization}/{self.name}/OWNERS"
 
+    def get_release_tag(self):
+        return f"{self.organization}-{self.name}-{self.version}"
+
+    def check_index(self, index):
+        """Check if the chart is present in the Helm index
+
+        Args:
+            index (dict): Content of the Helm repo index
+
+        Raise:
+            HelmIndexError if:
+            * The provided index is malformed
+            * The Chart is already present in the index
+
+        """
+        try:
+            chart_entry = index["entries"].get(self.name, [])
+        except KeyError as e:
+            raise HelmIndexError(f"Malformed index {index}") from e
```

Comment:
If I'm reading this right, we'd never see a key error here because you're returning a default value.

```python3
>>> x = {}.get("foo", [])
>>> x
[]
```